cluster-autoscaler/cloudprovider/upcloud/vendor.sh

```
@@ -0,0 +1,77 @@
+    echo "Usage: $0 <package> <package version>"
+    exit 1
+fi
+
+sdk_dir=`dirname $0`/pkg/${UPCLOUD_SDK_PACKAGE}/upcloud
+sdk_url=https://raw.githubusercontent.com/UpCloudLtd/upcloud-go-api/${UPCLOUD_SDK_VERSION}
+
+mkdir -p $sdk_dir/client $sdk_dir/service $sdk_dir/request
+
+sdk_download () {
+    echo "${2} => ${1}"
+    curl -sO --output-dir $1 $2
```

Shellcheck (0.11.0)

[info] Double quote to prevent globbing and word splitting.

(SC2086)